src/main/java/org/fundacionjala/coding/nestor/Main.java

```diff
@@ -0,0 +1,10 @@
+package org.fundacionjala.coding.nestor;
+
+public class Main {
+    public static void main(String[] args) {
+        Customer customer = new Customer("Test");
+        customer.addRental(new Rental(new Movie("The Revenant", 1), 2));
+        customer.addRental(new Rental(new Movie("Terminator", 0), 2));
+        System.out.println(customer.statement());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)